ci/infra/testrunner/testrunner.py

```diff
@@ -74,7 +74,8 @@ def node_upgrade(options):
 
 def test(options):
     TestDriver(options.conf, options.platform).run(test_suite=options.test_suite, test=options.test,
-                                                   verbose=options.verbose, collect=options.collect)
+                                                   verbose=options.verbose, collect=options.collect,
+                                                   skip_setup=options.skip_setup)
```

Comment:
This change being 2 lines is why I really prefer the closing brace/paren to be on a line by itself. ;)

Reply:
I really dislike trailing commas not sure why 😂

Reply:
Probably because they often break lesser languages. :p

Reply:
Check it out, never forget a comma again! ;)
```
run(test_suite=options.test_suite, test=options.test
                                                 , verbose=options.verbose
                                                 , collect=options.collect
                                                 , verbose=options.verbose
                                                 , collect=options.collect
                                                 , skip_setup=options.skip_setup
                                                 )
```

Reply:
Augh, my eyes! :p